### PR TITLE
Feat.button.moreactions

### PR DESCRIPTION
**Pull request by jeremie-lucca**
https://github.com/LuccaSA/lucca-front/issues/647

image

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://797-pr-lucca-front-LuccaSA.surge.sh

File: packages/scss/src/components/_button.scss [hunk 1]
```diff
@@ -123,6 +123,19 @@
...
 	}
 
+	&.mod-more {
+		&::before {
+			content: "";
+			@include makeIcon("arrow_south");
+			position: absolute;
+			top: 0;
+			left: 0;
+			font-size: .7rem;
```
Comment (lucienbertin) [inline packages/scss/src/components/_button.scss]: does this work with the different sizes of buttons

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://797-pr-lucca-front-LuccaSA.surge.sh

File: packages/scss/src/components/_button.scss (hunk 1 above)
Comment (sanlucca) [reply packages/scss/src/components/_button.scss]: And you are setting top/left twice (0 then .75.rem)

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://797-pr-lucca-front-LuccaSA.surge.sh

**Pull request merged**